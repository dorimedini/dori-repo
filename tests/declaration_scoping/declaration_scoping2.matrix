{
	int x=1,y;
	{ 
		print x;
		int x=3;
		
		if (true) 
			int x=2;
		print x;
		
		if (true) 
		{
			int x=22;
			print x;
			print y;
		}
		print x;
	}
	print x;
	print y;
}