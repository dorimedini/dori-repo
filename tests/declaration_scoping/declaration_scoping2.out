1
3
22
0
3
1
0
